newdle/client/src/components/answer/Calendar.js

```diff
@@ -97,17 +99,28 @@ function getAnswerProps(slot, answer) {
   }
 }
 
-function getSlotProps(slot, duration, minHour, maxHour, answer) {
+function getSlotProps(slot, duration, minHour, maxHour, answer, newdleTz, userTz) {
   const start = toMoment(slot, DEFAULT_FORMAT);
   const end = toMoment(start).add(duration, 'm');
   const answerProps = getAnswerProps(slot, answer);
   const height = calculateHeight(start, end, minHour, maxHour);
-  const pos = calculatePosition(start, minHour, maxHour);
+
+  let startMomentForTimezone = toMoment(slot, DEFAULT_FORMAT, newdleTz).tz(userTz);
```

Comment:
`...ForTimezone` sounds a bit strange. In Indico we usually use "local" to indicate times in a local timezone, e.g. `startMomentLocal`. Or `startMomentUserTz` / `startMomentUser`? `...Local` sounds least strange to me though.

Anyway, naming discussions are not really important right now (and they tend to be harder than the actual technical discussions, because naming is a [hard problem](https://twitter.com/secretGeek/status/7269997868))

Reply:
Ok, I'll revisit the naming. I think startTimeLocal should work 👍

Reply:
Immplemented startTimeLocal and startMomentLocal